packages/calcite-components/src/components/block/block.tsx

```diff
@@ -303,7 +303,6 @@ export class Block
           <button
             aria-controls={regionId}
             aria-expanded={collapsible ? toAriaBoolean(open) : null}
-            aria-label={toggleLabel}
             class={CSS.toggle}
             id={buttonId}
             onClick={this.onHeaderClick}
```

Comment:
It seems the `title` attribute is conflicting with AT, but if we add an `aria-describedby` and an unique `id` using `guid` to the `header`, similar to the `buttonId` and `regionId`, we could provide both contexts.

Something like:

```ts
const headerId = `${guid}-header`;

...

<header class={CSS.header} id={headerId}>

...

<button
...
aria-describedby={headerId}
...
>


```